### PR TITLE
Upgrade NumPy to 1.12.0 (resolves #2589)

### LINKED ISSUE #2589: Upgrade NumPy to 1.12.0

### PR DESCRIPTION
**Pull request by 0x00b1**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -40,8 +39,10 @@ virtualenv:
 install:
   - pip install --upgrade pip
   - pip install --upgrade wheel
+  - pip install --upgrade numpy
   - pip install --upgrade pytest
   - pip install --upgrade cython
+  - pip install --editable git+https://github.com/CellProfiler/centrosome.git#egg=centrosome
```
Comment (mcquin) [inline .travis.yml]: Are there changes needed for centrosome + numpy 1.12?

File: cellprofiler/modules/crop.py [hunk 2]
```diff
@@ -225,33 +225,33 @@ def create_settings(self):
...
-            other modules. </p>
-            <p><i>Important note:</i> Plate Fix uses the coordinates
-            entered in the boxes normally used for rectangle cropping (Top, Left and
-            Bottom, Right) to tighten the edges around your identified plate. This
-            is done because in the majority of plate identifications you do not want
-            to include the sides of the plate. If you would like the entire plate to
-            be shown, you should enter "1:end" for both coordinates. If, for example, you would like
-            to crop 80 pixels from each edge of the plate, you could enter Top, Left and Bottom,
-            Right values of 80 and select <i>%(FROM_EDGE)s</i>.</p>""" % globals())
+        # self.use_plate_fix = cps.Binary(
```
Comment (mcquin) [inline cellprofiler/modules/crop.py]: Can plate fix be removed entirely?

**Comment by mcquin** [comment]
Nice!!

File: .travis.yml (hunk 1 above)
Comment (mcquin) [inline .travis.yml]: Nevermind! I see the commits. :)

File: cellprofiler/modules/crop.py (hunk 2 above)
Comment (0x00b1) [inline cellprofiler/modules/crop.py]: Yes!

**Pull request merged**